### PR TITLE
https://issues.redhat.com/browse/ACM-14676

### PR DESCRIPTION
**Pull request by oafischer**
2.12, 2.11, and 2.10

### CODE REVIEW

File: networking/submariner/subm_configure.adoc [hunk 1]
```diff
@@ -52,6 +52,38 @@ View the following table to see the Submariner ports that you must enable:
...
+[source,yaml]
+----
+annotations:
+  submariner.io/control-plane-sg-id: <control_plane_group_id> <1>
+  submariner.io/subnet-id-list: <subnet_id_list> <2>
+  submariner.io/vpc-id: <custom_vpc_id> <3>
+  submariner.io/worker-sg-id: <worker_security_group_id> <4>
+----
++
+<1> Replace with your control plan security group ID. Typically, the name is similar to `<infra-id>-master-sg`.
```
Comment (aswinsuryan) [inline networking/submariner/subm_configure.adoc]: This description I gave created confusion for the users. it should mean this  "Typically, the name is similar to `<infra-id>-master-sg` for the security group we need to take the ID of that". They thought they need to use name here , where as it is supposed to be ID.

File: networking/submariner/subm_configure.adoc [hunk 2]
```diff
@@ -52,6 +52,38 @@ View the following table to see the Submariner ports that you must enable:
...
+<2> Replace with a comma-separated list of public subnet IDs in your custom VPC.
+<3> Repalce with your custom VPC ID.
+<4> Replace with your worker security group ID. Typically, the name is similar to `<infra-id>-worker-sg`.
```
Comment (aswinsuryan) [inline networking/submariner/subm_configure.adoc]: Same comment as above here.

File: networking/submariner/subm_configure.adoc (hunk 1 above)
Comment (oafischer) [reply networking/submariner/subm_configure.adoc]: I see, how does the following sound then?

"Typically, you can find the ID as part of the following name: `<infra-id>-master-sg`"

Comment (aswinsuryan) [reply networking/submariner/subm_configure.adoc]: > I see, how does the following sound then?
> 
> "Typically, you can find the ID as part of the following name: `<infra-id>-master-sg`"

 something like this "Typically, you can find the ID from the security group which has the following name `<infra-id>-master-sg`"

**Comment by oafischer** [comment]
Updated, if you want to take another look @aswinsuryan. Thanks!

**Review by aswinsuryan — APPROVED** [review]

File: networking/submariner/subm_configure.adoc [hunk 3]
```diff
@@ -52,6 +52,38 @@ View the following table to see the Submariner ports that you must enable:
...
+[source,yaml]
+----
+annotations:
+  submariner.io/control-plane-sg-id: <control_plane_group_id> <1>
+  submariner.io/subnet-id-list: <subnet_id_list> <2>
+  submariner.io/vpc-id: <custom_vpc_id> <3>
+  submariner.io/worker-sg-id: <worker_security_group_id> <4>
+----
++
+<1> Replace with your control plane security group ID. Typically, you can find the ID from the controle plane security group that has a name similar to `<infra-id>-master-sg>`.
```
Comment (dockerymick) [inline networking/submariner/subm_configure.adoc]:
```suggestion
<1> Replace with your control plane security group ID. Typically, you can find the ID from the control plane security group that has a name similar to `<infra-id>-master-sg>`.
```

**Review by dockerymick — APPROVED** [review]
Left a suggestion for spelling. Otherwise, this looks good to me!

**Pull request merged**